regsync.yaml

```
@@ -210,6 +210,9 @@ sync:
 - source: coredns/coredns:1.9.4
   target: docker.io/rancher/mirrored-coredns-coredns:1.9.4
   type: image
+- source: coredns/coredns:v1.12.3
```

```suggestion
- source: coredns/coredns:1.12.3
```

version also does not contain the `v` char